safe-room.lic

```diff
@@ -72,6 +72,9 @@ class SafeRoom
   def wait_for_healing
     Flags.reset('healthy')
     Flags.reset('moved')
+    
+    lie_down
+    wait_for_script_to_complete('tendme')
```

Comment:
Yes. It will exit when there are no more bleeders. Line 35 calls "exit if bleeders.empty?"